libraries/AP_BLHeli/AP_BLHeli.cpp

```diff
@@ -151,12 +151,23 @@ const AP_Param::GroupInfo AP_BLHeli::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("RVMASK",  12, AP_BLHeli, channel_reversed_mask, 0),
 
+#if HAL_WITH_ESC_TELEM
+    // @Param: EXTLM
+    // @DisplayName: Enable extended BLHeli telemetry
+    // @Description: When set to 1, this listens for 13-byte extended telemetry packets from BLHeli ESCs
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Advanced
+    // @RebootRequired: True
+    AP_GROUPINFO("EXTLM",  13, AP_BLHeli, extended_telemetry, 0),
```

Comment:
an OPTIONS bitmask would be better

Reply:
This PR has changes related to DroneCAN ESCX implementation only. BLHeli implementation is removed from this PR.